### ISSUE TITLE
undefined method error for "process_in_background" when using with resque

### ISSUE THREAD

**Issue by tpletcher** [jstorimer/delayed_paperclip#52]
trying to add delayed_paperclip to existing functional paperclip installation. i use resque for background tasks and deploy on heroku, so am using foreman locally and a procfile to start-up processes, etc. 

when i add the process_in_background to my model that has the paperclip attachment, and then fire up the server i get a resque exception indicating that the rake task failed due to process_in_background being an undefined method. 

any help appreciated. i;m hoping running in the background will alleviate another issue i have with paperclip related to dynamic styles and the timing of the instantiation of the attributes....

**Comment by hendragunz** [comment]
Yeah.... I got this error too... I run my app on rails 3.1.3 + mongoDB (using mongoid)
When run "rake spec", there is raise error "undefined method process_in_background" in my ClassModel... 

Need some help.... :-)

**Comment by jrgifford** [comment]
Did you guys ever figure this out? (I haven't used mongoid in concert with delayed_paperclip)

**Comment by calicoder** [comment]
Ping. I'm having the same issue.. has anyone been successful using delayed_paperclip with mongoid paperclip?

**Comment by jrgifford** [comment]
@calicoder I don't think this was officially supported on https://github.com/jrgifford/delayed_paperclip/, which is the new upstream.
